src/containers/Footer/footer.scss

```diff
@@ -46,7 +46,10 @@
     letter-spacing: 0em;
     line-height: 21px;
     text-align: left;
-    text-decoration: none;
+
+    &::after {
```

Comment:
What would be the purpose to this style?

Reply:
This prevents the external link icon from displaying.  I will add a comment.